utils/utils.go

```diff
@@ -138,6 +138,52 @@ func HandlePanic() {
 	}
 }
 
+// GenerateGithubActions generates a GitHub Actions workflow file for Keploy
+func GenerateGithubActions(logger *zap.Logger, path string, appCmd string) {
+	// Determine the path based on the alias "keploy"
+	logger.Info("Determining the path of the keploy binary based on the alias \"keploy\"")
+
+	keployPath := "/usr/local/bin/keploy" // Default path
+	aliasCmd := exec.Command("which", "keploy")
+	aliasOutput, err := aliasCmd.Output()
+	if err == nil && len(aliasOutput) > 0 {
+		keployPath = strings.TrimSpace(string(aliasOutput))
+	}
+	logger.Info("Path of the keploy binary determined successfully", zap.String("path", keployPath))
+	// Define the content of the GitHub Actions workflow file
+	actionsFileContent := `name: Keploy
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Test-Report
+        uses: keploy/testgpt@main
+        with:
+          working-directory: ./
+          keploy-path: ` + keployPath + `
+          command: ` + appCmd + `
+`
+
+	// Define the file path where the GitHub Actions workflow file will be saved
+	filePath := ".github/workflows/keploy.yml"
```

Comment:
do not add it directly to the workflow, keep it in a seperate directory so that the user has the choice to add it to their workflow or not.

Reply:
Refactored to /github-actions/keploy.yml